src/interfaces/wallet.h

```diff
@@ -14,6 +14,7 @@
 #include <functional>
 #include <map>
 #include <memory>
+#include <psbt.h>
```

Comment:
2bc10122874421ba7ce379325ebc2420955b1b9b

I think you should forward declare what's necessary.

Reply:
I tried, but it results in having to import stuff all over the place, and errors like this:
```
interfaces/wallet.cpp:354:20: error: no matching function for call to 'FillPSBT'
            return FillPSBT(m_wallet.get(), psbtx, complete, sighash_type, sign, bip32derivs);
                   ^~~~~~~~
./wallet/psbtwallet.h:27:28: note: candidate function not viable: cannot convert argument of incomplete type 'interfaces::PartiallySignedTransaction' to 'PartiallySignedTransaction &' for 2nd argument
NODISCARD TransactionError FillPSBT(const CWallet* pwallet,
                           ^
1 error generated.
```